common/static/common/js/discussion/utils.js

```diff
@@ -184,10 +184,8 @@
             if (!params.error) {
                 params.error = function() {
                     self.discussionAlert(
-                        gettext('Sorry'),
-                        gettext(
-                            'We had some trouble processing your request. Please ensure you have copied any ' +
-                            'unsaved work and then reload the page.')
+                        gettext('Error'),
```

Comment:
My understanding was that these were two arguments: the title of the alert, and then the message. I decided to make the title say "Error" as it didn't seem that the title was optional. Let me look at this again.